src/BasePlatform.ts

```diff
@@ -478,4 +478,11 @@ export default abstract class BasePlatform {
             policyUri: config.privacy_policy_url,
         };
     }
+
+    /**
+     * Suffix to append to the `state` parameter of OIDC /auth calls, will be round-tripped to the callback URI
+     */
+    public getOidcClientState(): string {
+        return "";
```

Comment:
Is this empty string desired?

Reply:
Yeah, only the ElectronPlatform actually needs any data roundtripped like this